### PR TITLE
cleaned up two unit tests

### PR DESCRIPTION
**Pull request by austin007008**

### CODE REVIEW

File: warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java [hunk 1]
```diff
@@ -57,68 +61,92 @@ public void extractSelectorsLuceneQuery4() {
...
-        List<IntRange> useSplitRanges = extractor.parseUseSplitsRanges("0-2");
-        Assert.assertTrue(extractor.useSplit(useSplitRanges, 0));
-        Assert.assertTrue(extractor.useSplit(useSplitRanges, 1));
-        Assert.assertTrue(extractor.useSplit(useSplitRanges, 2));
-        Assert.assertFalse(extractor.useSplit(useSplitRanges, 3));
+        List<IntRange> useSplitRanges = extractor.parseUseSplitsRanges("0");
+        //  @formatter:off
+        assertAll(
+                        () -> {assertTrue(extractor.useSplit(useSplitRanges, 0));},
+                        () -> {assertFalse(extractor.useSplit(useSplitRanges, 1));}
```
Comment (apmoriarty) [inline warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java:74]: I think splits 2 and 3 got missed here

File: warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java [hunk 2]
```diff
@@ -57,68 +61,92 @@ public void extractSelectorsLuceneQuery4() {
...
+                        () -> {assertFalse(extractor.useSplit(useSplitRanges, 3));},
+                        () -> {assertTrue(extractor.useSplit(useSplitRanges, 4));}
+        );
+        //  @formatter:on
     }
 
     @Test
     public void rangeTest4() {
         SplitSelectorExtractor extractor = new SplitSelectorExtractor();
-        List<IntRange> useSplitRanges = extractor.parseUseSplitsRanges("2,4,6-");
```
Comment (apmoriarty) [inline warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java:94]: Add this back?

**Review by apmoriarty — APPROVED** [review]

File: warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java (hunk 1 above)
Comment (austin007008) [reply warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java:74]: Some of the test got reordered and that's why the diff shows some pieces missing or changed. all of them are still here.

File: warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java (hunk 2 above)
Comment (austin007008) [reply warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java:94]: see comment above

File: warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java (hunk 1 above)
Comment (apmoriarty) [reply warehouse/query-core/src/test/java/datawave/audit/SplitSelectorExtractorTest.java:74]: Perfect, thanks!

**Review by MiguelRicardos — APPROVED** [review]

**Pull request merged**